app/pathogen/arbovirus/dashboard/filters.tsx

```diff
@@ -29,64 +53,77 @@ const addFilterMulti = (
   });
 };
 
+// Function to build a filter dropdown
 const buildFilterDropdown = (
   filter: string,
   placeholder: string,
   state: ArboContextType,
   filterOptions: string[],
   map: mapboxgl.Map | null | undefined,
-  data: any,
+  data: any
 ) => {
-
-
   return (
-      <div className="pb-3">
-        <MultiSelect
-          handleOnChange={(value) =>
-            addFilterMulti(value, filter, state, map, data)
-          }
-          heading={placeholder}
-          selected={state.selectedFilters[filter] ?? []}
-          options={filterOptions.filter((assay: string) => assay != null)}
-        />
-      </div>
+    <div className="pb-3">
+      <MultiSelect
+        handleOnChange={(value) =>
+          addFilterMulti(value, filter, state, map, data)
+        }
+        heading={placeholder}
+        selected={state.selectedFilters[filter] ?? []}
+        options={filterOptions.filter((assay: string) => assay != null)}
+      />
+    </div>
   );
 };
 
+// Main Filters component
 export default function Filters(props: { map?: mapboxgl.Map | null }) {
+  // Get the state and map from the context and props
   const state = useContext(ArboContext);
   const { map } = props;
 
+  // Fetch arbovirus data using the useArboData hook
   const { data } = useArboData();
 
+  // Fetch filter options using React Query
   const filters = useQuery({
     queryKey: ["ArbovirusFilters"],
     queryFn: () =>
       fetch(`${process.env.NEXT_PUBLIC_API_BASE_URL}/arbo/filter_options`).then(
-        (response) => response.json(),
+        (response) => response.json()
       ),
   });
 
+  const resetFilters = () => {
+    // Dispatch action to reset filters
+    state.dispatch({
+      type: ArboActionType.RESET_FILTERS,
+      payload: {}, // Include an empty object as payload
+    });
+  };
+
   if (filters.isSuccess && !filters.isLoading && !filters.isError) {
     console.debug(filters.data, Object.keys(filters.data));
 
     return (
       <div>
         <div className="p-0">
-          {/*<div>*/}
-          {/*    <SectionHeader*/}
-          {/*        header_text={"Demographics"}*/}
-          {/*        tooltip_text={"Participant related data"}*/}
-          {/*    />*/}
-          {/*</div>*/}
+          <div>
```

Comment:
Can we make the filter bar scrollable so that filters do not get cut off when the screen size is small

![image](https://github.com/PathoTracker/Pathotracker/assets/29795835/eae3eb19-1519-466f-bf6f-d3695fdad086)

Reply:
![Screenshot 2023-12-01 at 16-12-13 Pathotracker](https://github.com/PathoTracker/Pathotracker/assets/59937930/59b8c068-a5f9-436a-a89e-2c701678bef7)

Reply:
added a scrollbar so that in smaller screen the user can scroll down